evil-collection-git-timemachine.el

```diff
@@ -0,0 +1,22 @@
+;;; evil-collection-git-timemachine.el --- Bindings for `git-timemachine' -*- lexical-binding: t -*-
+;; Author: William Carroll <wpcarro@gmail.com>
+
+;;; Commentary:
+;; Evil keybindings for `git-timemachine' that conform to the principles outlines in evil-collection
+
+;;; Code:
+(require 'evil)
+(require 'git-timemachine)
+
+(defun ecgt--setup-bindings ()
+  "Setup `git-timemachine' since `evil-set-initial-state' is unavailable to
+minor modes."
+  (evil-motion-state nil)
```

Comment:
Instead of this, set the initial state in the setup function.

```
(evil-set-initial-state 'free-keys-mode 'normal)
```

^ free-keys example.

We prefer normal state to motion.

Reply:
Hmnn, totally glossed over the fact you said you tried this already.

Reply:
Hmnn, it's probably not needed either way. The mode will probably start up in normal state so you don't have to do anything special other than bind the keys in normal state.